README.md

```diff
@@ -132,3 +132,19 @@ End-to-end testing is developed with Robot Framework; this higher-lever layer of
 - demonstrate *User Stories* from high level perspective
 - demonstrate coherent logical data mapping by performing the same high level capabilities, using REST API flow Vs Python client flow,
 directly checking the end results in the backend gRPC MLMD server.
+
+## Known Issues
```

Comment:
```suggestion
## Tips
```

this is not an issue, this is a known behaviour of the docker.io container registry